### DISCUSSION TITLE
doc: Betting web standards? Or did you mean bettering or backing?

### DISCUSSION THREAD

**Discussion by corneliusroemer** [mswjs/msw#2337] [Ideas]
I came across this line and was wondering what was meant by "Betting" here:

https://github.com/mswjs/msw/blob/562fac96fb5ba1da950bba170ecb05f5156be938/decisions/jest-support.md?plain=1#L5

3 possible meanings I can think of:

- “Bettering and advancing the web standards is one of the goals behind this project.”
- “Backing and advancing the web standards is one of the goals behind this project.”
- “Betting on advancing web standards is one of the goals behind this project.”

**Comment by kettanaito** [comment]
Hi, @corneliusroemer. Thanks for raising this. I agree, the wording is confusing. I meant it this way:

> Betting on and advancing the web standards ...

Does this make more sense?

**Reply by corneliusroemer** [reply]
Sure!

**Reply by kettanaito** [reply]
Fixed in 6702d778c541ebae09d5799490f645b57a12c01f.
